src/Illuminate/Support/Str.php

```diff
@@ -356,6 +356,19 @@ public static function finish($value, $cap)
         return preg_replace('/(?:'.$quoted.')+$/u', '', $value).$cap;
     }
 
+    /**
+     * Convert a value to flat case.
+     *
+     * @param  string  $value
+     * @return string
+     */
+    public static function flat($value)
+    {
+        $string = static::replace(['-', '_', ' '], '', $value);
```

Comment:
What would be with a "Hi, there!"? I don't think that "hi,there!" is a valid flatcase. You'd better to use snake/slug/smth. under the hood.

Reply:
I think using the `slug` method with a custom `$separator` would implement flatcase correctly.

```
return Str::slug('Hi, there!', ''); // 'hithere'
```